tests/src/testklshell.cpp

```diff
@@ -136,14 +138,16 @@ static auto NonLinearKLShellLoadControlTR() {
   vtkWriter->setFileNamePrefix("TestKLShell");
   vtkWriter->setFieldInfo("Displacement", Dune::VTK::FieldInfo::Type::vector, 3);
 
-  auto lc = Ikarus::LoadControl(tr, 1, {0, 1});
+  auto lc = LoadControl(tr, 1, {0, 1});
   lc.subscribeAll(vtkWriter);
   const auto controlInfo = lc.run();
 
   t.check(controlInfo.success);
-  std::cout << std::setprecision(16) << std::ranges::max(d) << std::endl;
-  t.check(Dune::FloatCmp::eq(0.2087577577980777, std::ranges::max(d)))
-      << std::setprecision(16) << "The maximum displacement is " << std::ranges::max(d);
+  const auto maxDisp = std::ranges::max(d);
+  std::cout << std::setprecision(16) << maxDisp << std::endl;
+  t.check(Dune::FloatCmp::eq(0.2087574597947082, maxDisp, 1e-6))
```

Comment:
Can you also add a test `checkFEByAutoDiff` as available in `tests/src/testcommon.hh` ? may be it makes the test slow then? It can also be a single element test?